README.md

```diff
@@ -111,9 +100,10 @@ A few constants are automatically defined. These are *_RAMSTART*, *_EXTSTART*, a
 .extra_memory 512
 ```
 
-**.function**: Adds a function header to the output file. The directive may be immediately followed by ```stk``` to specify that the arguments to this function should be passed on the stack rather than copied into the local variables (see the [glulx spec](https://www.eblong.com/zarf/glulx/glulx-spec_1.html#s.6.2) for details). This is followed by the local variable specification. Locals may be specified by either a positive integer (which will create that many unnamed local variables to be accessed by index) or with a list of local variable names. These styles may not be combined, but if a function has no local variables the specification may be omitted entirely.
+**.function**: Adds a function header to the output file. The directive may be immediately followed by `stk` to specify that the arguments to this function should be passed on the stack rather than copied into the local variables (see the [glulx spec][glulx spec 6.2] for details). This is followed by the local variable specification. Locals may be specified by either a positive integer (which will create that many unnamed local variables to be accessed by index) or with a list of local variable names. These styles may not be combined, but if a function has no local variables the specification may be omitted entirely.
```

Comment:
It may just be my unfamiliarity with a lot of Markdown syntax, but is there a reason you've got two [glulx spec] in there? Only one actually seems to show up in the final result.